<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContactPage</class>
 <widget class="QWidget" name="AddContactPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加联系人</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../浏览器下载内容/chat.png</normaloff>../../浏览器下载内容/chat.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>账号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameAddr">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="publicKeyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>公钥:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="publicKeyAddr">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="backupLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>备注:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="backup">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectUserButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPublicKeyButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
